matcher/vf2.py

```diff
@@ -105,7 +106,7 @@ def candidate_pairs_iter(self):
             # checkme: process the valid node pairs with the closest embedding first
             ordered_T1_inout = sorted(
                 T1_inout.copy(),
-                key=lambda u: self.comparator(self.e1[u], self.e2[node_2])
+                key=lambda u: -self.cosf(self.e1[u], self.e2[node_2])
```

Comment:
This could be improved with a top K search?

Reply:
Yes